src/gsy_e/gsy_e_core/simulation/external_events.py

```diff
@@ -0,0 +1,51 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of Grid Singularity Exchange.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
```

Comment:
Nit: I found out that it's usually better to use comments for licences, and not docstrings. This is because license is not code documentation, and docstrings end up being part of the module documentation. You can see it for example like this:
```py
from gsy_e.gsy_e_core import simulation

print(simulation.__doc__)
```
This should print something that describes the content of the `simulation` module. Instead, right now it prints the license of GSy (and it will do the same for all modules, which makes this documentation unusable).
What I'd suggest for now and for the future is to use normal comments with `#`.

Reply:
OK, thanks for the info. We should discuss and decide about this in the team. Does not make sense, that I change this here in the few modules. It should be changed globally at once IMO. So How about a short discussion on slack and a ticket that come out of it?

Reply:
Sure, I can bring this on slack!